graphql/version_resolver.go

```diff
@@ -302,26 +302,25 @@ func (r *versionResolver) Tasks(ctx context.Context, obj *restModel.APIVersion,
 			taskSorts = append(taskSorts, task.TasksSortOrder{Key: key, Order: order})
 		}
 	}
-	v, err := model.VersionFindOne(model.VersionById(versionId).WithFields(model.VersionRequesterKey))
+	baseVersionID := ""
+	baseVersion, err := model.FindBaseVersionForVersion(utility.FromStringPtr(obj.Id))
 	if err != nil {
-		return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding version with id: '%s': %s", versionId, err.Error()))
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding base version id for version with id: '%s': %s", versionId, err.Error()))
```

Comment:
nit. shorten error message & remove the first colon
```suggestion
		return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding base version id for version '%s': %s", versionId, err.Error()))
```